nf_core/modules/modules_utils.py

```diff
@@ -201,7 +201,7 @@ def prompt_module_version_sha(module, modules_repo, installed_sha=None):
     git_sha = ""
     page_nbr = 1
 
-    all_commits = modules_repo.get_module_git_log(module)
+    all_commits = modules_repo.get_component_git_log(module, "modules")
```

Comment:
```suggestion
    all_commits = modules_repo.get_component_git_log(module,self.component_type)
```

Reply:
This doesn't have access to `self` or `component_type` - it's a helper function in`module_utils`. Unless you are meaning to do this only if we move the method?

Reply:
ah, sorry, looked like it was in the same context as the one above. 👍🏻